napari/layers/base/base.py

```diff
@@ -4,20 +4,19 @@
 from xml.etree.ElementTree import Element, tostring
 import numpy as np
 from skimage import img_as_ubyte
+from ._constants import Blending
 
 from ...components import Dims
-from ...util.event import Event
+from ...util.event import EmitterGroup, Event
 from ...util.keybindings import KeymapMixin
-from ._visual_wrapper import VisualWrapper
+from ...util.status_messages import status_format
 
 
-class Layer(VisualWrapper, KeymapMixin, ABC):
+class Layer(KeymapMixin, ABC):
```

Comment:
check if we use abstract methods any more. if not, you can remove `ABC`

Reply:
we do - `data` is still an abstract method